### PR TITLE
Switch from travis to ghactions

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

**Comment by johanandren** [comment]
(Already updated required statuses/checks)

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
+          fetch-depth: 0
+
+      - name: Set up JDK 8
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.8.0
```
Comment (patriknw) [inline .github/workflows/build-test.yml:19]: is this sample still on 8?

Comment (johanandren) [reply .github/workflows/build-test.yml:19]: I missed that the travis build was on 11, I'll update.

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**